.pre-commit-config.yaml

```diff
@@ -13,7 +13,7 @@ repos:
   rev: '6.0.0'
   hooks:
   - id: flake8
-    exclude: repository_service_tuf_api/__init__.py|venv|.venv|setting.py|.git|.tox|dist|docs|/*lib/python*|/*egg|build|tools
+    exclude: docs
```

Comment:
Removing `repository_service_tuf_api/__init__.py` from the `exclude` list means we would lint `repository_service_tuf_api/__init__.py` with `flake8` from now on as `repository_service_tuf_api/__init__.py` is not ignored in `.gitignore`.
That's what we want, correct?

Reply:
It seems to be a legit python file. I don't see a reason why we won't lint it.